### PR TITLE
Webpack style export

### PR DESCRIPTION
**Pull request by kevinkace**
Adds option to disable `styleExports` in Webpack bundle.

## Description
Adds a check before adding `var style = ...` to the Webpack bundle.

## Motivation and Context
This value is not always needed, and removing it can reduce the bundle size.

## How Has This Been Tested?
Added a test. Checked the output. Defaults to the current behavior.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by tivac** [comment]
Idea is good, I've got some more specific feedback that I'll add when I'm not on my phone.

File: packages/webpack/loader.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ module.exports = async function(source) {
             out.push(`export var ${ident} = ${JSON.stringify(exported[ident])};`);
         });
 
-        out.push(`export var styles = ${JSON.stringify(result.details.result.css)};`);
+        if(options.styleExport !== false) {
```
Comment (tivac) [inline packages/webpack/loader.js]: Can this be added to options as a default with a value of `true`, so this check could then be `if(!options.styleExport)`?

Comment (kevinkace) [reply packages/webpack/loader.js]: I though adding `styleExport : true` to [`processor.js`](https://github.com/tivac/modular-css/blob/master/packages/processor/processor.js#L46) would set the default, but not seeing the expected result.

Comment (tivac) [reply packages/webpack/loader.js]: `styleExport` isn't something the processor knows anything about, that's a bundler-specific thing that should probably go into [`plugin.js`](https://github.com/tivac/modular-css/blob/39153d76186e567eaaf3f8e57e2c9beb90d8aa0b/packages/webpack/plugin.js#L19-L22).

Something like this, probably:

```js
 var options = Object.assign(
    Object.create(null),
    { styleExport : true },
    args
);
```

(or broken out into `const defaults = { ... }` or something & then mixed into the `options`)

Comment (kevinkace) [reply packages/webpack/loader.js]: Aha, ok. Thanks.

Comment (kevinkace) [reply packages/webpack/loader.js]: Still working on this, maybe.

**Comment by tivac** [comment]
Aww crap @kevinkace I led you astray. For some reason I though the Plugin/Loader shared options but they 100% don't.

**Comment by kevinkace** [comment]
Ahh ok thanks! I can keep digging to see where a default can go.

**Comment by kevinkace** [comment]
A little ugly, but looks like the recommended way to add defaults with [loader-utils](https://github.com/webpack/loader-utils).

**Comment by tivac** [comment]
I'll cut a release in a bit here, thanks for sticking with it!

**Comment by tivac** [comment]
Took me longer than hoped for, but this is now released as `@modular-css/webpack@24.1.0`. Thanks again for all your hard work @kevinkace!

**Comment by kevinkace** [comment]
Updating our builds rn! thanks

**Pull request merged**